Exchange-Guide.md

```diff
@@ -0,0 +1,196 @@
+# Exchange Guide (WIP)
+
+## Goal
+This guide describes how to setup `xud` with an existing exchange system and how to connect to the XUD Simulation Network, `xud-simnet` for short and do some test trades. `xud` is in alpha stage and the `xud-simnet` aims to give exchange operators a 'feel' of how things will work. `xud` in it's current stage should never be connected to a production system nor be used on mainnet. You'll be using test coins at all times. This guide purposely is kept relatively manual to make it understandable how `xud` functions. We are working on an improved setup using docker, which will also be the preferred way to use `xud` for exchanges in production.
+
+Once the setup of `xud-simnet` is completed, you will be able to query pending orders, place orders, and experiment with `xud`'s rich set of commands. Once a match is found in the network, your orders will be executed via cross-chain atomic swaps. `xud-simnet` currently supports atomic swaps between BTC and LTC. This guide will show how to to install GO, bitcoin and litecoin daemons, setup lightning daemons (LND) for bitcoin (BTC) and litecoin (LTC), setup `xud` and basic instructions of how to integrate its API and finally connect to the `xud-simnet`. This guide is written for Linux & MacOS operating systems and geared towards IT administrators of exchanges.
```

Comment:
I never see golang stylized as "GO" - just "Go" would be better. And instead of "setup `xud` and basic instructions of how to integrate its API" (which kind of sounds like we're setting up the instructions) I'd write "setup `xud`, integrate with its API, and finally..."

Reply:
done